asset-transfer-basic/chaincode-typescript/Dockerfile

```diff
@@ -16,7 +16,7 @@ ARG CC_SERVER_PORT
 
 # Setup tini to work better handle signals
 ENV TINI_VERSION v0.19.0
-ENV PLATFORM=amd64
+ENV PLATFORM=arm64

```

Comment:
For most people this should still be amd64 right?

Reply:
Whoops!   Yes thank you for the catch.  That accidentally bled over from some local testing.   The correct technique to specify a target platform in a build is to use the `TARGETARCH` and `TARGETOS` build ARGs set by buildkit.  These ARGs will be set correctly for `buildx` when generating cross-platform docker images.

I have reverted this back to amd64, keeping this PR focused ONLY on the new sample. I'll open a different PR to track improvements on the M1 builds.